src/Godot/Nativescript.hs

```diff
@@ -317,7 +437,9 @@ getSingleton = do
     Just o -> tryCast o
     Nothing -> do
       ge <- getEngine
-      o <- get_singleton ge =<< toLowLevel (convertClassName name)
+      -- FIXME ANDREI I don't understand what convertClassName is doing here? It seems wrong.
```

Comment:
The Godot class name is prefixed with an underscore, but that is impossible in Haskell. Ergo we need to reintroduce it to access the singleton from Godot

Reply:
That's what I thought too, but when I reintroduce that line of code I get an error with:
`(Just os) <- getSingleton @OS`
I wonder if somehow this conversion is happening twice or Godot no longer prefixes with underscores?